### ISSUE TITLE
dynamic camera settings

### ISSUE THREAD

**Issue by chriscarex** [xml3d/xml3d.js#182]
Hi,
I would like to change the camera behaviour dynamically.
After loading the initial camera settings:

`function onload() {
    var viewElement = document.getElementById("geometry");
    var cam = new XML3D.StandardCamera(viewElement, {mode: "examine", zoomSpeed: 200});
    cam.examine(viewElement);
  }`

I would like to change the value of zoomSpeed.
I tried severa lmethods, such as initialising a new camera, updating the camera with 
`cam = new XML3D.StandardCamera(viewElement, {mode: "examine", zoomSpeed: 100});`
using global variables replacing the number, but none of them seem to work when applied dynamically.
Any suggestion on how to solve this issue? Is there a way to stop the first camera and initialise a new one?
Thanks!

**Comment by csvurt** [comment]
Hi,

you should be able to set it directly on the camera object:

```javascript
var cam = new XML3D.StandardCamera(viewElement, {mode: "examine", zoomSpeed: 200}); 
cam.examine(viewElement); 
...
cam.zoomSpeed = 100;
```

If you re-initialize a camera you should always detach the old one first, otherwise all the event listeners will still be bound:

```javascript
cam.detach();
cam = new XML3D.StandardCamera(viewElement);
```
